examples/pytest/Readme.md

```diff
@@ -0,0 +1,26 @@
+# How to run this examples
+
+1. Clone the repository
+
+```bash
+git clone https://github.com/qase-tms/qase-python.git
+```
+
+2. Move to the directory with the examples
+
+```bash
+cd qase-python/examples/pytest
+```
+
+3. Install the required packages
+
+```bash
+pip install -r requirements.txt
+```
+
+4. Add the Qase token and project code to the `qase.config.json` file
+5. Run the tests
+
+```bash
+pytest
```

Comment:
shouldn't we add `--qase-mode=testops`?

Reply:
No, because it is specified in the configuration file